tests/gnrc_rpl_p2p/tests/01-run.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+from testrunner import run
+
+def testfunc(child):
```

Comment:
Extra newline required according to `flake8`.

Reply:
pushed a fixup